<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetInfo</class>
 <widget class="QWidget" name="AssetInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollArea {
border: 0px solid green;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="asset_name">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Asset Name</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="asset_versionSelect">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">QLabel[type=&quot;0&quot;] {
	color: rgb(120, 120, 120);
	font-size: 10px;
	margin-bottom: 0;
}

QLabel[type=&quot;1&quot;] {
	margin-bottom: 4px;
}

QScrollBar:vertical
    {
        background-color:rgb(68, 68, 68);
        width: 10px;
        margin: 0px 3px 0px 3px;
        border: 1px transparent #2A2929;
        border-radius: 4px;
    }

    QScrollBar::handle:vertical
    {
        background-color: rgb(86, 86, 86);
        min-height: 5px;
        border-radius: 4px;
    }

QScrollBar::sub-line:vertical
{
	 border: none;
	background: none;
}

QScrollBar::add-line:vertical
{
	border: none;
	background: none;
 }

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
	 border: none;
     background: none;
     color: none;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>312</width>
          <height>297</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="asset_description">
           <property name="text">
            <string>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent eget volutpat velit, ut vulputate ligula. Donec placerat nulla vitae dolor ultrices, et mollis enim mattis. Ut egestas quam ex, quis tincidunt arcu tempus vel. Nullam dictum, enim nec tristique porttitor, ante ligula volutpat neque, sed fringilla ligula purus eget risus. Donec velit nisi, dictum ac suscipit vel, tincidunt eget lorem. Fusce arcu turpis, suscipit at convallis eu, efficitur at urna. Curabitur lorem tortor, blandit a metus sit amet, lobortis hendrerit massa. Suspendisse molestie justo vitae malesuada commodo. Fusce euismod leo id arcu semper, at finibus velit fermentum.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
           <property name="type" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_changes">
           <property name="text">
            <string>Changes</string>
           </property>
           <property name="type" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="asset_changes">
           <property name="text">
            <string>- shome changes</string>
           </property>
           <property name="type" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_createdDate">
           <property name="text">
            <string>Created</string>
           </property>
           <property name="type" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="asset_createdDate">
           <property name="text">
            <string>16 Sep 2020</string>
           </property>
           <property name="type" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_tags">
           <property name="text">
            <string>Tags</string>
           </property>
           <property name="type" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="asset_tags">
           <property name="text">
            <string>tagA, tagB, C</string>
           </property>
           <property name="type" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
